jeongjaein/FBMember/app/src/main/java/user/manage/fbmember/ModifyActivity.kt

```diff
@@ -1,4 +1,15 @@
 package user.manage.fbmember
 
-class ModifyActivity {
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+import kotlinx.android.synthetic.main.activity_modify.*
+class ModifyActivity : AppCompatActivity(){
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_login)
```

Comment:
```suggestion
        setContentView(R.layout.activity_ modify)
```
ModifyActivity에 login이?

Reply:
수정하겠습니당

Reply:
헉 커밋이 수정사항이 다 반영이안되고 올라가서 그런가봐용 수정한걸로 다시올릴게용